pkg/controller/controller_test.go

```diff
@@ -392,6 +426,25 @@ func newBPWithKopiaSnapshotOutputArtifact() *crv1alpha1.Blueprint {
 	}
 }
 
+func newBPForProgressRunningPhase() *crv1alpha1.Blueprint {
+	return &crv1alpha1.Blueprint{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: "test-bp-running-phase-",
+		},
+		Actions: map[string]*crv1alpha1.BlueprintAction{
+			"backup": {
+				// set output artifacts from main phases as well as deferPhase
+				OutputArtifacts: map[string]crv1alpha1.Artifact{},
+				Phases: []crv1alpha1.BlueprintPhase{
+					*phaseWithNameAndCMD("backupPhaseOne", []string{"sleep", "8"}),
+					*phaseWithNameAndCMD("backupPhaseTwo", []string{"sleep", "5"}),
+				},
+				DeferPhase: phaseWithNameAndCMD("deferPhase", []string{"sleep", "5"}),
```

Comment:
```suggestion
					*phaseWithNameAndCMD("backupPhaseOne", []string{"sleep", "10"}),
					*phaseWithNameAndCMD("backupPhaseTwo", []string{"sleep", "8"}),
				},
				DeferPhase: phaseWithNameAndCMD("deferPhase", []string{"sleep", "8"}),
```
Increase sleep to avoid flakes